protos/flyteidl/admin/agent.proto

```diff
@@ -36,7 +43,7 @@ message CreateTaskRequest {
   core.TaskTemplate template = 2;
   // Prefix for where task output data will be written. (e.g. s3://my-bucket/randomstring)
   string output_prefix = 3;
-  // subset of task execution metadata that are relevant to out-core plugins
+  // subset of runtime task execution metadata that are relevant to external plugins
```

Comment:
Not sure whether `external` is a more official name.

Reply:
I think we officially decided that this feature was going to be called "flyte agents". The notion of "out-of-core plugins" means that nothing will be compiled into propeller, which this is not fixing for existing plugins, and the "external plugins" was not descriptive enough.

Reply:
As I understood, `agent` refers to the routing plugin https://github.com/flyteorg/flyteplugins/tree/master/go/tasks/plugins/webapi/agent, but here it refers to the receiving part or the RPC server. Or maybe I misunderstood that, it is the RPC server being agent.

Reply:
Agent is a Python gRPC server that can run certain tasks (BQ, snowflake, etc). agent plugin (https://github.com/flyteorg/flyteplugins/tree/master/go/tasks/plugins/webapi/agent) is actually a gRPC client, which sends the request to the agent.